IdentityCore/src/cache/ios/MSIDKeychainTokenCache.m

```diff
@@ -518,7 +518,12 @@ - (BOOL)saveWipeInfoWithContext:(id<MSIDRequestContext>)context
 
     MSID_LOG_WITH_CTX_PII(MSIDLogLevelInfo, context, @"Full wipe info: %@", MSID_PII_LOG_MASKABLE(wipeInfo));
     
-    NSData *wipeData = [NSKeyedArchiver archivedDataWithRootObject:wipeInfo];
+    NSData *wipeData;
+#if TARGET_OS_UIKITFORMAC
+    wipeData = [NSKeyedArchiver archivedDataWithRootObject:wipeInfo requiringSecureCoding:YES error:nil];
```

Comment:
should be use "msid" NSKeyedArchiver utilities here as well?

Reply:
done